app/views/sequence/new.html.erb

```diff
@@ -15,12 +47,24 @@
   </div>
   <div>
     <label>Type</label>
-    <input v-model="network.selected.step.type" />
+    <select v-model="network.selected.step.type">
+      <option value="">Please select</option>
+      <option>EventHandler</option>
+      <option>WebRequest</option>
+      <option>JsonParser</option>
+      <option>Trigger</option>
+      <option>EventFormatter</option>
+      <option>Post</option>
+    </select>
```

Comment:
A dropdown of our event types... obv we can automate this in some way in the future.